### DISCUSSION TITLE
No transparency shown on custom system

### DISCUSSION THREAD

**Discussion by FabianGolembiewski-sp** [AvaloniaUI/Avalonia#16708] [Q&A]
Hi, I need some help regarding transparency.

We have a Customer using Windows 10 Build 14393 in a Citrix terminal server environment.
Our app is using transparent background areas, but they are not shown. We have configured TransparencyBackgroundFallback to LimeGreen, and this color is shown.

The question is, has anybody an idea how to configure Windows 10, to test this "TransparencyBackgroundFallback"?
I tried to disabled all kind of transparency and effects in Windows, but the transparency is still shown.

Thanks for any kind of help.

**Comment by kekekeks** [comment]
1) Use Transparent instead of blurbehind, we don't support blur for win10 < 17134
2) Check if DirectComposition is active via Spy++ (the window should have WS_EX_NOREDIRECTIONBITMAP extended style)

**Comment by FabianGolembiewski-sp** [comment]
I now have a virtual machine using Windows 10 Build 14393.

The window does not have this style.
![image](https://github.com/user-attachments/assets/f7561cc4-71ec-45f2-b434-d2937b6836da)

I will change our application to not use transparent areas (rounded borders), as a Hotfix.
